rsconnect/bundle.py

```diff
@@ -94,12 +94,10 @@ def __init__(
                 "version": quarto_inspection.get("quarto", {}).get("version", "99.9.9"),
                 "engines": quarto_inspection.get("engines", []),
             }
-            project_config = quarto_inspection.get("config", {}).get("project", {})
-            render_targets = project_config.get("render", [])
-            if len(render_targets):
-                self.data["metadata"]["primary_rmd"] = render_targets[0]
```

Comment:
Do we not need to set `primary_rmd`?

Reply:
It's not used by Connect for Quarto content, today.

With R Markdown and Jupyter content, we tell the tool the specific file that we want to render, and the output file is written to the `.index` file.

This path only applies for projects, and we want to always rely on the Quarto discovery rules in that case. Only projects have the `config.project.render` structure in the output from `quarto inspect`.

For Quarto standalone documents, there is a stronger case for communicating the primary document/entrypoint. Connect currently transforms content into a project and then has Quarto use its default discovery rules. If the primary file is deployed along with secondary Markdown files, those would need to be named `_filename.md` to be ignored during rendering. Otherwise, Quarto could render both `primary.qmd` and `do-not-render.qmd`. They could also deploy a `_quarto.yml` file alongside the file, and deploy it as a simple "project". Both of those options feel OK, right now.

We may want to eventually annotate the manifest with `metadata.entrypoint`, but I'd like to see if we can avoid it.